app/controllers/katello/concerns/smart_proxies_controller_extensions.rb

```diff
@@ -5,10 +5,23 @@ module SmartProxiesControllerExtensions
 
       included do
         append_view_path('app/views/foreman')
-        before_action :find_resource_and_status, :only => [:pulp_storage, :pulp_status]
+        before_action :find_resource_and_status, :only => [:capsule_puppet_path, :pulp_storage, :pulp_status]
         alias_method_chain :action_permission, :katello
         alias_method_chain :show, :content
 
+        def capsule_puppet_path
```

Comment:
This method is nearly identical to the `pulp_storage` one below, could we make it DRY'er?